src/envs/pddl_procedural_generation.py

```diff
@@ -110,6 +112,9 @@ def _blocks_piles_to_strs(
     return all_strs
 
 
+################################## Delivery ###################################
+
+
```

Comment:
nit: seems like other similar comments (Blocks and Forest) leave one newline after the comment, whereas this leaves 2. Can we standardize the number of newlines before and after this header comment?